### PR TITLE
Ba1 and ba2 omicron sublineages

### LINKED ISSUE #30: present data for BA1 & BA2 omicron sub-lineages
BA.1 has both the HV 69/70 deletion and the K417N mutation
BA.2 contains the K417N mutation, but not the HV 69/70 deletion

### PR DESCRIPTION
**Pull request by ashleyasmus**
this should close #30

### CODE REVIEW

File: metc-wastewater-covid-monitor/wastewater-monitoring-report.Rmd [hunk 1]
```diff
@@ -0,0 +1,564 @@
+---
```
Comment (eroten) [inline metc-wastewater-covid-monitor/wastewater-monitoring-report.Rmd:1]: Should this be in the metc-wastewater-covid-monitor/ directory?

**Review by eroten — CHANGES_REQUESTED** [review]
Nice job! I love the comments throughout the new R scripts. 

- Lets finesse the new plots a bit before we finish up. The plot captions no longer fit 
- Is the wastewater-monitoring-report.Rmd used in the app at all? If not, we should move it to the parent directory so it doesn't get uploaded when we deploy the app

**Comment by eroten** [comment]
Also, I'm getting this error when the app is run 

image

File: metc-wastewater-covid-monitor/wastewater-monitoring-report.Rmd (hunk 1 above)
Comment (ashleyasmus) [reply metc-wastewater-covid-monitor/wastewater-monitoring-report.Rmd:1]: i agree! this should be in its own directory.

**Review by ashleyasmus — COMMENTED** [review]
i still want to re run figures to make sure BA.2 is NA before we announce it's all public but this looks great, thank you for the merge conflict resolve

**Comment by eroten** [comment]
Great! Thanks Ashley!

Could you double check that R/d_load.R runs for you? I'm getting a rename error when I try to run it. I checked out the repo on my PC and its consistent across Mac and PC. 


```
Error: Can't rename columns that don't exist.
x Column `t1_copies_u_l` doesn't exist.
```

**Comment by ashleyasmus** [comment]
ready for one more check @eroten ?

**Comment by eroten** [comment]
Could you double check the column names in server.R? The table is getting both `copies_day_person_M_mn` and `copies_day_person_7day`. Should it be one or the other?

**Comment by ashleyasmus** [comment]
i think both load columns are okay - but we needed them labeled! fixed now.

**Review by eroten — APPROVED** [review]

**Review by ashleyasmus — COMMENTED** [review]
nice!

**Pull request merged**